### DISCUSSION TITLE
4G LTE and 5G Network Native Cloud Telecom Applications on Kubernetes Infrastructure and the related

### DISCUSSION THREAD

**Discussion by opentelcokloud** [cncf/telecom-user-group#100] [General]
Hello Team,

I have been using Rancher/ Kubernetes and the related tools: Prometheus/ Grafana/ Helm/ EFK/ ... as the Infrastructure to install the Telecom Applications on to it. The result is quite good: Rancher & Kubernetes help to install/ manage/ monitor/ orchestrate my (Open5gs) LTE EPC/ 5G 5GC CNF and the (srs-RAN) eNB-UE quite comfortably.

1. I installed the Open5GS: 4G EPC Packet Core and 5GC CNF Apps on to my Kubernetes Cluster via Rancher’s
2. Then I installed SRS-LTE eNB-UE on to the same Kubernetes Cluster
    I got the eNB-UE established connection with EPC then Internet

Kubernetes Infas and the mentioned tools also help with other essential operational tasks via its functionalities: Dashboard, Performance Monitoring and Logging and ++

I recorded some of the results in the Youtube link below:
https://www.youtube.com/channel/UCdrZzzEgfXbsPfnvovHez9w/featured

Please join me if you are intested in this

**Comment by taylor** [comment]
Hello, @opentelcokloud. 

Would you be interested in presenting on this at the next TUG meeting?

It sounds like you may have some use cases which can be added to the [Cloud native Network Function Working Group (CNF WG)](https://github.com/cncf/cnf-wg) either directly with a PR or added to the discussion board. There are probably good practices we can identify from the use cases and your experience which is the primary goal for the CNF WG.

**Reply by opentelcokloud** [reply]
Thank you Taylor for your reply.
Yes, I would love to.
But, I would like to know more about what is needed for that presentation? Is it like just to share what we did about our interested topics (like what I demonstrated in my videos) or more formal material/ protocols/results/ suggestions are expected?
And any other details so that I could prepare better for it
